### PR TITLE
Identify and fix PHPunit static methods called in non-static way in tests

### PR DESCRIPTION
**Pull request by ktzouno**

### CODE REVIEW

File: src/InterNations/Sniffs/Waste/PhpUnitStaticallyCalledMethodsSniff.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    public function register(): array
+    {
+        return [T_OBJECT_OPERATOR, T_DOUBLE_COLON];
+    }
+
+    /** @param integer $stackPtr */
+    public function process(CodeSnifferFile $file, $stackPtr): void
+    {
+        $tokens = $file->getTokens();
+        $classNameToken = static::findCurrentClassNameToken($file, $stackPtr, $tokens);
```
Comment (CMalvika) [inline src/InterNations/Sniffs/Waste/PhpUnitStaticallyCalledMethodsSniff.php]: if you are passing file, you can get $tokens, without passing it.

Comment (ktzouno) [inline src/InterNations/Sniffs/Waste/PhpUnitStaticallyCalledMethodsSniff.php]: True, solved (y)

File: src/InterNations/Sniffs/Waste/PhpUnitStaticallyCalledMethodsSniff.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                static::findClassMethods(TestCase::class, [ReflectionMethod::IS_PUBLIC, ReflectionMethod::IS_STATIC]);
+        }
+
+        if (!in_array($nextToken['content'], static::$testCasePublicStaticMethods, true)) {
+            return;
+        }
+
+        $currentToken = $tokens[$stackPtr];
+
+        $file->addFixableError(
```
Comment (CMalvika) [inline src/InterNations/Sniffs/Waste/PhpUnitStaticallyCalledMethodsSniff.php]: Why use of FixableError? in stead addError?

File: src/InterNations/Sniffs/Waste/PhpUnitStaticallyCalledMethodsSniff.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+            'Call PHPUnit methods statically, replace %s%s%s() with self::%s()',
+            $stackPtr,
+            'StaticallyCallPhpUnitMethods',
+            [$previousToken['content'], $currentToken['content'], $nextToken['content'], $nextToken['content']]
+        );
+
+        $file->fixer->beginChangeset();
```
Comment (CMalvika) [inline src/InterNations/Sniffs/Waste/PhpUnitStaticallyCalledMethodsSniff.php]: Amazing..! Is this working already, or we need any extra fixer library?

**Pull request merged**